async/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Urgh yeah I remember this : `WorkerParameters` is `final` and has a restricted constructor. I think it'd be good to have an experiment with the worker manager testing tools (https://developer.android.com/topic/libraries/architecture/workmanager/how-to/testing-worker-impl) to see if we can use that instead of introducing `final` mocking to the module.

Reply:
Yeah we can use it. Done.